app/helpers/action_links_helper.rb

```diff
@@ -65,6 +65,9 @@ def display_refund_link_for?(resource)
     # Display the link only if there is at least one payment of the relevant type
     return false if resource.payments.find { |payment| payment.payment_type == "GOVPAY" }.nil?
```

Comment:
I think it would be safer not to hardcode payment types and use convenience methods like payment.govpay? or payment.refund?

`return false if resource.payments.find { |payment| payment.payment_type == "GOVPAY" }.nil?`
=>
`return false if resource.payments.find { |payment| payment.govpay? }.nil?`